frontend/src/components/maps/leaflet/Legend/LegendControl.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../../../styles.scss';
+
+.legend-button {
+  background-color: #ffffff !important;
+  color: #121212 !important;
+  width: 40px !important;
+  height: 40px !important;
+  font-size: 25px !important;
+  display: flex !important;
+  align-items: center !important;
+}
```

Comment:
Why the need for `!important` here? 
Isn't it a better way to override the styles? i.e. 
```css
.LegendControl .legend-button {
  ...
}
```

Reply:
Ideally we wouldn't use !important.